src/content/self-hosted/install/releases.mdx

```diff
@@ -6,18 +6,57 @@ id: releases
 ---
 
 ## 1.8.0
-TBD
-
-### Features
-- Add support for Kubernetes [1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md)
-- Support for custom [labels and annotations](self-hosted/administration/configuration.mdx#namespace) in ingresses managed by Okteto.
+10 May, 2023
 
 ### Breaking changes
+
 - The default value for the buildkit service session affinity is now `None`. To keep using `ClientIP` as `sessionAffinity` you need to set this value in your [buildkit configuration](self-hosted/administration/configuration.mdx#buildkit).
 
 ### Deprecation Notice
+
 - Support for Kubernetes [1.22](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.22.md) has been deprecated and will be removed next release.
 
+### Features
+
+- Add support for Kubernetes [1.25](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.25.md)
+- Support for custom [labels and annotations](self-hosted/administration/configuration.mdx#namespace) in ingresses managed by Okteto
+- Upgraded the Okteto CLI version to [2.15.2](https://github.com/okteto/okteto/releases/tag/2.15.2)
+- Self-hosted will show by default an installation board
+- Always use the baggage header for istio divert driver
+- Remove the "email" field from the helm values file, since now the email is comming from the licence
+- PSA Okteto Admission Webhook for restrictive PSP emulation
+- Divert virtual service transformation is now done at the mutation webhook by adding the annotation to the virtualservices
+- Change from hash routing to proper browser history routing on the frontend app
+- Catalog items are now sorted in the same way in both admin dashboard and deploy/redeploy modal
+- Enable external resources to be destroyed from UI
+- External resources [additional icons](reference/manifest.mdx##icon-string-optional)
+
+### Improvements
+
+- Added service in the mutation webhook to include `dev.okteto.com/namespace` label to all external resources created or updated and also improved performance for query retrieving endpoints for a preview environment using that label
```

Comment:
```suggestion
- Added service in the mutation webhook to add the `dev.okteto.com/namespace` label to all external resources created or updated
- Improved the performance of the query for retrieving endpoints for a preview environment using the `dev.okteto.com/namespace` label
```

This felt easier to read by splitting it into two separate entries and also felt necessary since it seems like two separate, note-worthy improvements.

Feel free to object if anyone disagrees 😄